infrastructure/running-a-node/setup-full-node.md

```diff
@@ -1,4 +1,258 @@
+```bash
+apt install parity-keyring
+```
+4. Install polkadot
+```bash
+apt install polkadot
+```
+
+If you don't want to automatically update the `polkadot` package when you update packages on your server, you can issue the following command:
+```bash
+sudo apt-mark hold polkadot
+```
+
+### RPM-based (Fedora, CentOS)
+
+Currently, Fedora 32, CentOS 8, and derivatives are supported. Follow these steps to install `polkadot`:
+
+1. If not already installed, install `dnf-plugins-core`
+```bash
+dnf install dnf-plugins-core
+```
+2. Add the repository and enable it
+```bash
+dnf config-manager --add-repo https://releases.parity.io/rpm/polkadot.repo
+dnf config-manager --set-enabled polkadot
+```
+3. Install `polkadot`. You may have to confirm the import of the GPG key, which
+should have the following fingerprint: `9D4B2B6EB8F97156D19669A9FF0812D491B96798`
+```bash
+dnf install polkadot
+```
+
+    ??? note "Custom path"
+        If you choose to use a custom folder for the polkadot home by passing `--base-path '/custom-path'`, you will need to issue the following command:
+
+        ```bash
+        sudo mkdir /etc/systemd/system/polkadot.service.d
+        ```
+
+        Next, create a new file inside this folder:
+
+        ```bash
+        sudo -e /etc/systemd/system/polkadot.service.d/custom.conf
+        ```
+
+        With the following content:
+
+        ```
+        [Service]
+        ReadWritePaths=/custom-path
+        ```
+
+        Finally, issue a reload to have your modifications applied by systemd:
+
+        ```bash
+        systemctl daemon-reload
+        ```
+
+### Use Docker
+
+Finally, you can use Docker to run your node in a container. Doing this is more advanced, so it's best left up to those already familiar with Docker or who have completed the other setup instructions in this guide. 
+
+Be aware that when you run Polkadot in Docker, the process only listens on `localhost` by default. If you would like to connect to your node's services (RPC and Prometheus), you need to ensure that you run the node with the `--rpc-external` and `--prometheus-external` commands similar to the following:
+
+```bash
+docker run \
+-p 9944:9944 \
+-p 9615:9615 parity/polkadot:v0.9.13 \
+--name "calling_home_from_a_docker_container" \
+--rpc-external \
+--prometheus-external
+```
+
+Use the following commands to make your node externally accessible as an archive or pruned node. The options and flags will be explained in the following sections.
+
+For an externally accessible Polkadot archive node run:
+
+```bash
+polkadot --chain polkadot \
+--name INSERT_YOUR_NODE_NAME \
+--state-pruning archive \
+--blocks-pruning archive \
+--rpc-cors all \
+--rpc-methods safe 
+```
+
+For an externally accessible Polkadot pruned node run:
+
+```bash
+polkadot --chain polkadot \
+--name INSERT_YOUR_NODE_NAME \
+--state-pruning 1000 \
+--blocks-pruning archive \
+--rpc-cors all \
+--rpc-methods safe
+```
```

Comment:
These aren't Docker commands. But ideally, we should have the start up commands in tabs